libc/docs/full_host_build.rst

```diff
@@ -76,6 +84,13 @@ above.
 Build and install
 =================
 
+.. warning::
```

Comment:
seems like we should add this warning message to our cmake.

Reply:
I've added a TODO